server/lib/payExpense.js

```diff
@@ -0,0 +1,46 @@
+const config = require('config');
+const Promise = require('bluebird');
+const uuid = require('node-uuid');
+
+module.exports = app => {
+  const services = {
+    paypal: (groupId, expense, email, preapprovalKey) => {
+      var uri = `/groups/${groupId}/transactions/${expense.id}/paykey/`;
+      var baseUrl = config.host.webapp + uri;
+      var amount = expense.amount;
+      var payload = {
+        requestEnvelope: {
+          errorLanguage: 'en_US',
+          detailLevel: 'ReturnAll'
+        },
+        actionType: 'PAY',
+        currencyCode: expense.currency.toUpperCase() || 'USD',
+        feesPayer: 'SENDER',
+        memo: `Reimbursement transaction ${expense.id}: ${expense.description}`,
```

Comment:
If the user will see this memo, no point in printing the txnId. It won't mean anything to the user. If group name can be had, I'd rather print that. `Reimbursement from ${group.name}: ${expense.description}`